playbooks/chameleon_usage.yml

```diff
@@ -14,6 +14,10 @@
       when: enable_usage_appliance_report
       vars:
           action: 'heat_template'
+    - role: chameleon_usage
+      when: enable_usage_report_sync
```

Comment:
We need to add a default for this, sort of like https://github.com/ChameleonCloud/chi-in-a-box/blob/68b2d551c4489841ebe6ff6eaf3746c457fbc124/kolla/defaults.yml#L145. Otherwise this will fail with a "variable not defined" error.